packages/sdk/ts/utils/utils.ts

```diff
@@ -0,0 +1,35 @@
+import os from "os";
+
+declare global {
+  interface ITerminatable {
+    terminate: () => Promise<unknown>;
+  }
+
+  // eslint-disable-next-line vars-on-top, no-var, camelcase
+  var curve_bn128: ITerminatable | undefined;
+
+  // eslint-disable-next-line vars-on-top, no-var, camelcase
+  var curve_bls12381: ITerminatable | undefined;
```

Comment:
Is it possible to use `curve_bn128?: ITerminatable`?

Reply:
no that doesn't work